Modules/CraftQueue/CraftQueue.lua

```diff
@@ -289,6 +289,11 @@ function CraftSim.CRAFTQ:ImportRecipeScan()
         for _, recipeData in pairs(filteredRecipes) do
             local restockOptions = CraftSim.CRAFTQ:GetRestockOptionsForRecipe(recipeData.recipeID)
             local restockAmount = CraftSim.DB.OPTIONS:Get("CRAFTQUEUE_GENERAL_RESTOCK_RESTOCK_AMOUNT")
+
+            if CraftSim.DB.OPTIONS:Get("TSM_RESTOCK_KEY_ITEMS") ~= "" then

```

Comment:
is an empty string recognized as a valid tsm expression in the edit box? if yes this might be a problem when trying to remove it again. If not we may need to add a little info icon to the restock input to inform players of this or otherwise they will wonder how to disable this again